lib/kubernetes-deploy/runner.rb

```diff
@@ -131,6 +138,8 @@ def run(verify_result: true, allow_protected_ns: false, prune: true)
       success = false
     ensure
       @logger.print_summary(success)
+      tags = %W(namespace:#{@namespace} sha:#{ENV['REVISION']})
```

Comment:
Also this was liberate since `@current_sha` was set in `runner`and I wanted to keep things identical in all calling places